lib/Sema/TypeCheckDecl.cpp

```diff
@@ -676,8 +676,9 @@ IsStaticRequest::evaluate(Evaluator &evaluator, FuncDecl *decl) const {
       dc->isTypeContext()) {
     const auto operatorName = decl->getBaseIdentifier();
     if (auto ED = dyn_cast<ExtensionDecl>(dc->getAsDecl())) {
-      decl->diagnose(diag::nonstatic_operator_in_extension,
-                     operatorName, ED->getExtendedTypeRepr())
+      decl->diagnose(diag::nonstatic_operator_in_extension, operatorName,
+                     ED->getExtendedTypeRepr() != NULL,
```

Comment:
Please use `nullptr` or `(bool) ED->getExtendedTypeRepr()`

Reply:
Fixed. BTW: What is the convention here for resolving conversations?

Reply:
There is no convention really, this is just mostly used as an organisational tool. Usually, I resolve a conversation if someone addressed my suggestion or I myself have addressed a suggestion as an author, given there is some confidence that the conversation has exhausted itself and the underlying issue/idea/etc is trivial enough to fold away from the direct line of sight of other reviewers.